cli/package.json

```diff
@@ -70,6 +70,7 @@
     "lodash": "4.17.10",
     "log-symbols": "2.2.0",
     "minimist": "1.2.0",
+    "moment": "2.22.2",
```

Comment:
ok, challenge is how to keep this `moment` version in sync with actual bundled version of `moment`, but should be close enough to just grab types